extensions/odoo_theme/static/scss/_sphinx_basic.scss

```diff
@@ -598,23 +598,15 @@ ol.upperroman {
 //     margin-bottom: 0px;
 // }
 
-ol.simple ol p,
-ol.simple ul p,
-ul.simple ol p,
-ul.simple ul p {
-    margin-top: 0;
-}
-
-ol.simple > li:not(:first-child) > p,
-ul.simple > li:not(:first-child) > p {
-    margin-top: 0;
-}
-
 ol.simple p,
 ul.simple p {
     margin-bottom: 0;
 }
 
+li > .simple {
+    margin-bottom: 1em;
+}
+
```

Comment:
We could go further, I'd use `.simple` as the main selector and use the power of SCSS 🙂 
```scss
.simple {
    p {
        margin-bottom: 0;
    }
    
    li > & {
        margin-bottom: 1em;
    }
}
```
But, again, is the `simple` class used on other elements than `ul` & `ol` ? @AntoineVDV maybe you know it? 🙂

Reply:
No idea^^

Reply:
@Brieuc-brd `.simple` is also used on `dl` elements. [This SCSS](https://github.com/odoo/documentation/pull/7515#discussion_r1486018508) works well for the `dl` element too. 

I searched through the documentation and did not find any other instances of where the class `.simple` is used on other elements. This [docutils Examples of Syntax Constructs](https://docutils.sourceforge.io/docs/user/rst/demo.html#table-of-contents) document also only shows the class being used on only the `ul`, `ol`, and `dl` elements.

The `dl`, `dt`, and `dd`  elements are heavily used on the `applications/finance/accounting/get_started/cheat_sheet.rst` doc [for example](https://www.odoo.com/documentation/15.0/applications/finance/accounting/get_started/cheat_sheet.html#accounting-cheat-sheet).

I will push up a commit to update the CSS to SCSS since it is a more elegant solution 👍